e2e/journey-suites/src/device-profile.test.ts

```diff
@@ -9,32 +9,39 @@ import { expect, test } from '@playwright/test';
 import { asyncEvents } from './utils/async-events.js';
 import { username, password } from './utils/demo-user.js';
 
-test.skip('Test happy paths on test page', async ({ page }) => {
+test('Test device profile collection journey flow', async ({ page }) => {
   const { clickButton, navigate } = asyncEvents(page);
-  await navigate('/?journey=TEST_DeviceProfile');
 
   const messageArray: string[] = [];
 
-  // Listen for events on page
   page.on('console', async (msg) => {
     messageArray.push(msg.text());
     return Promise.resolve(true);
   });
 
-  // Perform basic login
+  await navigate('/?journey=DeviceProfileCallbackTest&clientId=basic');
+
+  await expect(page.getByLabel('User Name')).toBeVisible({ timeout: 10000 });
   await page.getByLabel('User Name').fill(username);
   await page.getByLabel('Password').fill(password);
   await clickButton('Submit', '/authenticate');
 
-  await expect(page.getByText('Collecting device profile')).toBeVisible();
+  await expect(page.getByText('Collecting device profile information...')).toBeVisible({
+    timeout: 10000,
+  });
+  await expect(page.getByText('Device profile collected successfully!')).toBeVisible({
+    timeout: 15000,
+  });
```

Comment:
I don't mind testing for these messages, indirectly testing Device Profile, but I think we need to more directly test the feature by grabbing the request body, parsing it and ensuring the JSON at least has the expected base properties with truthy values.